### PR TITLE
Scripting: Unlimited compilation rate for ingest

### PR DESCRIPTION
**Pull request by stu-elastic**
* `ingest` and `processor_conditional` default to unlimited compilation rate

Refs: #50152

### CODE REVIEW

**Comment by stu-elastic** [comment]
`./gradlew ':server:test' --tests "org.elasticsearch.snapshots.SnapshotResiliencyTests.testSnapshotWithNodeDisconnects" -Dtests.seed=5C27E5D78E6B69FF -Dtests.security.manager=true -Dtests.locale=en-CA -Dtests.timezone=GMT0 -Druntime.java=11` 

is failure in master.

See: https://gradle-enterprise.elastic.co/s/xg3l6wvjapram

**Comment by stu-elastic** [comment]
@elasticmachine test elasticsearch-ci/1

**Review by rjernst — APPROVED** [review]
LGTM

File: server/src/main/java/org/elasticsearch/script/ScriptCache.java [hunk 1]
```diff
@@ -51,15 +51,15 @@
...
+    final CompilationRate rate;
     private final double compilesAllowedPerNano;
     private final String contextRateSetting;
 
     ScriptCache(
-            int cacheMaxSize,
-            TimeValue cacheExpire,
-            Tuple<Integer, TimeValue> maxCompilationRate,
-            String contextRateSetting
+        int cacheMaxSize,
```
Comment (stu-elastic) [inline server/src/main/java/org/elasticsearch/script/ScriptCache.java:59]: Re-indent.

**Review by jdconrad — APPROVED** [review]

**Pull request merged**